### PR TITLE
Added integrations tutorial

### LINKED ISSUE #5: Include Elastic and Splunk integration guide
Hello!

We want to include a guide inside documentation to easily manage Elastic/OpenSearch and Splunk integration with FIM.

### PR DESCRIPTION
**Pull request by okynos**
Hello!

Coming from #5 and https://github.com/Achiefs/fim/issues/119 we have added integrations tutorial to configure FIM with network environment.

This PR closes #5 
Added:
- Splunk integration tutorial with images.
- ElasticSearch/OpenSearch tutorial.
- Added `token` parameter in configuration page.
- Added `max_file_checksum` parameter.
- Added `watcher` selector parameter.

Regards!

### CODE REVIEW

File: docs/integrations.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+---
+
+# Splunk integration configuration
+
+The Splunk integration is set by the `endpoint` configuration section. This section has different options. In addition, the Splunk integration will require additional steps to receive FIM events into the Splunk indexer. The steps are the following:
+
+First, it is required to configure the Splunk indexer to accept FIM events.
+
+1. At the Splunk home page, click on the `Settings` tab.
+![Splunk home page](../../assets/images/splunk/home.png)
```
Comment (Zenidd) [inline docs/integrations.md:59]: Screenshots are broken both in my local copy of the repo and on the online preview @okynos 
image

**Review by Zenidd — CHANGES_REQUESTED** [review]
Please fix the broken screenshots

**Comment by okynos** [comment]
Solved in https://github.com/Achiefs/documentation.achiefs.github.io/pull/6/commits/b03b4f7ab84419c4793b8e5ed2cb3057f6ac6c56
and https://github.com/Achiefs/documentation.achiefs.github.io/pull/6/commits/10160071eb4afb5fce88098af30fb6f149e9688f

The images works as expected in development environment. Unfortunately Jekyll works this way.
image

**Comment by Zenidd** [comment]
> The images work as expected in development environment. Unfortunately Jekyll works this way.


I see! Was expecting the .png file to be rendered in the MD in the local/Github preview. 

LGTM!

**Pull request merged**